src/main/java/optic_fusion1/mcantimalware/configuration/MemoryConfigurationOptions.java

```diff
@@ -17,17 +17,13 @@ public MemoryConfiguration configuration() {
         return (MemoryConfiguration) super.configuration();
     }
 
-    @NotNull
     @Override
-    public MemoryConfigurationOptions copyDefaults(boolean value) {
+    public void copyDefaults(boolean value) {

```

Comment:
Once again unneeded changes.
Previously this classed used the Builder Pattern, which you've removed